docs/drivers.rst

```diff
@@ -36,37 +36,178 @@ the ``lib/`` directory. Some drivers may not work without them.
+    MCP9808 Temperature <https://circuitpython.readthedocs.io/projects/mcp9808/>
+    MPL3115A2 Barometric Pressure, Altitude and Temperature Sensor <https://circuitpython.readthedocs.io/projects/mpl3115a2/>
+    SGP30 Air Quality <https://circuitpython.readthedocs.io/projects/sgp30/>
+    SHT31-D Temperature and Humidity <https://circuitpython.readthedocs.io/projects/sht31d/>
+    Si7021 Temperature and Humidity <https://circuitpython.readthedocs.io/projects/si7021/>
+    Thermistor Temperature <https://circuitpython.readthedocs.io/projects/thermistor/>
+
+Light Sensors
+---------------
+
+These sensors detect light related attributes such as ``color``, ``light`` (unit-less), and
+``lux`` (light in SI lux).
+
+.. toctree::
+
+    APDS9960 Proximity, Light, RGB, and Gesture <https://circuitpython.readthedocs.io/projects/apds9960/>
+    TCS34725 Color Sensor <https://circuitpython.readthedocs.io/projects/tcs34725/>
+    TSL2561 Light Sensor <https://circuitpython.readthedocs.io/projects/tsl2561/>
+    TSL2591 High Dynamic Range Light Sensor <https://circuitpython.readthedocs.io/projects/tsl2591/>
+    VCNL4010 Proximity and Light <https://circuitpython.readthedocs.io/projects/vcnl4010/>
+    VEML6070 UV Index <https://circuitpython.readthedocs.io/projects/veml6070/>
+
+Distance Sensors
+------------------
+
+These sensors measure the ``distance`` to another object and may also measure light level (``light`` and ``lux``).
+
+.. toctree::
+
+    VL6180x 5 - 100 mm <https://circuitpython.readthedocs.io/projects/vl6180x/>
+    VL53L0x ~30 - 1000 mm <https://circuitpython.readthedocs.io/projects/vl53l0x/>
+
+Radio
+--------
+
+These chips communicate to other's over radio.
+
+.. toctree::
+
+    RFM9x LoRa <https://circuitpython.readthedocs.io/projects/rfm9x/>
+    RFM69 Packet Radio <https://circuitpython.readthedocs.io/projects/rfm69/>
 
-Drivers
--------
+IO Expansion
+--------------
 
-Drivers provide easy access to sensors and other chips without requiring a
-knowledge of the interface details of the chip itself.
+These provide functionality similar to `analogio`, `digitalio`, `pulseio`, and `touchio`.
+
+.. toctree::
+
+    Adafruit SeeSaw <https://circuitpython.readthedocs.io/projects/seesaw/>
+    ADS1x15 Analog-to-Digital Converter  <https://circuitpython.readthedocs.io/projects/ads1x15/>
+    DS2413 OneWire GPIO Expander <https://circuitpython.readthedocs.io/projects/d2413/>
+    FocalTech Capacitive Touch <https://circuitpython.readthedocs.io/projects/focaltouch/>
+    MCP4725 Digital-to-Analog Converter <https://circuitpython.readthedocs.io/projects/mcp4725/>
+    PCA9685 16 x 12-bit PWM Driver <https://circuitpython.readthedocs.io/projects/pca9685/>
+    TLC5947 24 x 12-bit PWM Driver <https://circuitpython.readthedocs.io/projects/tlc5947/>
+    TLC59711 12 x 16-bit PWM Driver <https://circuitpython.readthedocs.io/projects/tlc59711/>
+    MCP4725 Capacitive Touch Sensor <https://circuitpython.readthedocs.io/projects/mpr121/>
```

Comment:
Sensor name or URL incorrect? Looks like sensor name is duplicated from above.

Reply:
Fixed